### PR TITLE
[INLONG-7070][Manager][DataProxy] Fix mq configuration dispatching problem

### LINKED ISSUE #7070: [Bug][Manager][DataProxy] Fix pulling MQ configuration problem
### What happened

Manager configuration is not correctly dispatched sometimes due to like dirty data.

### What you expected to happen

Correct configuration

### How to reproduce

Latest dp and manager

### Environment

_No response_

### InLong version

master

### InLong Component

InLong Manager, InLong DataProxy

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by woofyzhao**
- Fixes #7070

### CODE REVIEW

**Review by pocozh — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/repository/DataProxyConfigRepository.java [hunk 1]
```diff
@@ -385,7 +385,8 @@ private Map<String, List<InLongIdObject>> parseInlongId(Map<String, InlongGroupI
             InlongStreamId streamIdObj = entry.getValue();
             String groupId = streamIdObj.getInlongGroupId();
             InlongGroupId groupIdObj = groupIdMap.get(groupId);
-            if (groupId == null) {
+            if (groupId == null || groupIdObj == null) {
+                LOGGER.warn("groupId {} or groupIdObj {} is null, ignored", groupId, groupIdObj);
```
Comment (vernedeng) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/repository/DataProxyConfigRepository.java:389]: It is better to set logging level to **_debug_**.

Comment (woofyzhao) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/repository/DataProxyConfigRepository.java]: OK

File: inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/mq/MessageQueueZoneProducer.java [hunk 2]
```diff
@@ -107,6 +107,7 @@ public void run() {
      */
     public void reload() {
         try {
+            LOG.info("start reload mq clusters");
```
Comment (healchow) [inline inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/mq/MessageQueueZoneProducer.java:110]: This log seems has not enough info, will this log output too much?

Comment (woofyzhao) [reply inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/mq/MessageQueueZoneProducer.java:110]: no, low frequency

**Review by vernedeng — APPROVED** [review]
lgtm

**Pull request merged**